Tests/CoreDataBaseTestCase.m

```diff
@@ -18,16 +18,23 @@ -(void)buildPersistenceManagerWithDefaultClient:(BOOL)defaultClient {
     self.persistenceManager.classForAssets = [Asset class];
     self.persistenceManager.classForSpaces = [SyncInfo class];
 
-    NSArray* contentTypeIds = @[ @"1nGOrvlRTaMcyyq4IEa8ea", @"6bAvxqodl6s4MoKuWYkmqe",
-                                 @"6PnRGY1dxSUmaQ2Yq2Ege2", @"cat", @"test" ];
+    NSArray* contentTypeIds = @[ @"1nGOrvlRTaMcyyq4IEa8ea",
+                                 @"6bAvxqodl6s4MoKuWYkmqe",
+                                 @"6PnRGY1dxSUmaQ2Yq2Ege2",
+                                 @"cat",
+                                 @"test"
+                               ];
 
+    // FIXME: this is overriding content mappings done in super method call.
```

Comment:
Since you removed a fixme comment before, should we also remove this one and move it into github issues? (Same for below)